### PR TITLE
Add --match-versions (read versions from currently installed modules)

### PR DESCRIPTION
**Pull request by maxhq**

### CODE REVIEW

File: script/scan-prereqs-cpanfile [hunk 1]
```diff
@@ -74,6 +76,14 @@ if ($version) {
         $develop_prereqs = scan_test_requires($dir, $develop_prereqs);
     }
 
+    # Get minimum versions from currently installed modules (= available in @INC)
+    if ($match_versions) {
+		require V;
+        for my $prereqs ($runtime_prereqs, $configure_prereqs, $test_prereqs, $develop_prereqs) {
+            $prereqs = { map { $_ => V::get_version($_) } keys %$prereqs };
```
Comment (tokuhirom) [inline script/scan-prereqs-cpanfile]: I recommend to use Module::Info instead

Comment (maxhq) [inline script/scan-prereqs-cpanfile]: Yeah, sorry, I didn't look hard enough for a proper module.
But then I'd rather would like to use Module::Metadata which is a core module, any concerns about this?

**Comment by alexcrack** [comment]
Need this feature.
Please merge it.

**Comment by tokuhirom** [comment]
your pr is conflicting with other commits. could you resolve it?

**Comment by alexcrack** [comment]
> your pr is conflicting with other commits. could you resolve it?

This is not my PR :)
But I've fixed and created a new one.

**Comment by maxhq** [comment]
@alexcrack Thank you for waking me up ;-)

**Comment by alexcrack** [comment]
@tokuhirom, please merge)

**Comment by tokuhirom** [comment]
done!

**Comment by alexcrack** [comment]
Thank you!
Couldn't you update cpan module?

**Pull request merged**